index.bs

```diff
@@ -13519,9 +13522,14 @@ The following typographic conventions are used in this document:
 *   Grammar terminals: <emu-t>sometoken</emu-t>
 *   Grammar non-terminals: <emu-nt>ExampleGrammarNonTerminal</emu-t>
 *   Grammar symbols: <emu-t class="regex">identifier</emu-t>
-*   IDL types: {{unsigned long}}
-*   ECMAScript classes: {{ECMAScript/Map}}
+*   IDL types: <code class="idl">unsigned long</code> <!-- {{unsigned long}} -->
+*   IDL values: true, false, null, 0.42, −0
+*   ECMAScript classes: <code class="idl">Map</code> <!-- {{ECMAScript/Map}} -->
 *   ECMAScript language types: Object
+*   ECMAScript values: generally the same notation as IDL values; bolded type (e.g.
```

Comment:
I'm not sure we have the same perspective, here. I feel like ES values should be clearly marked as such across the spec, as they are in the ES spec.